### DISCUSSION TITLE
Duda enunciado

### DISCUSSION THREAD

**Discussion by Burbina200** [IIC2133-PUC/2024-2#7] [Tarea 0] [answered]
¡Hola!

Tengo una duda sobre el enunciado. En la parte 1, punto 2 (LEAVE ID), mencionan: "Es importante destacar que este pingüino siempre estará presente dentro de la tienda." ¿Podrían aclararme a qué se refieren con eso?

![Captura de pantalla 2024-08-13 160620](https://github.com/user-attachments/assets/ad1e7ff4-e661-47ef-9d9f-f548537d0c97)

**Comment by 4n1t4** [comment, marked as answer]
Se refiere a que al momento del evento el ID entregado siempre va a ser un pingüino que existe y se encuentra dentro de la tienda en ese instante, para que no deban considerar un caso borde.
